libs/types/src/constants/queue.constants.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ *  NOTE: Each service grouping (*-api, *-watcher) prefixes its BullMQ Redis connections with a service-specific
+ *        prefix, so these queue names, while common, will be unique per-service. Thus a queue "myMessageQueue" in
+ *        one service is distinct from a queue of the same name in another service.
+ *
+ *        Since the Gateway services are designed to be fully independent of one another, there should
+ *        be no reason to construct shared queue names across services.
+ */
+
+import { AnnouncementTypeName } from '#types/enums/announcement-type.enum';
+
+export namespace AccountQueues {
+  /**
+   * Name of the queue that publishes account transactions to Frequency blockchain
+   */
+  export const TRANSACTION_PUBLISH_QUEUE = 'transactionPublish';
+}
+
+export namespace ContentWatcherQueues {
+  /**
+   * Name of the queue that has all incoming requests
+   */
+  export const REQUEST_QUEUE_NAME = 'requestQueue';
+
+  /**
+   * All of the announcement type queues
+   */
+  export const BROADCAST_QUEUE_NAME = 'broadcastQueue';
+  export const REPLY_QUEUE_NAME = 'replyQueue';
+  export const REACTION_QUEUE_NAME = 'reactionQueue';
+  export const UPDATE_QUEUE_NAME = 'updateQueue';
+  export const TOMBSTONE_QUEUE_NAME = 'tombstoneQueue';
+  export const PROFILE_QUEUE_NAME = 'profileQueue';
```

Comment:
nit: These are also part of Content Publisher but not a big deal. Maybe we can merge Content Watcher and publisher later.

Reply:
Correct; since most of the content-watcher queues are also used in content-publishing, you'll see lower down that I import these into the content-publishing namespace, so at least we don't need to duplicate the values.